docs/en/datasets/recog.md

```diff
@@ -321,3 +328,31 @@ rm dataset.zip && rm -rf dataset
 ```bash
 python tools/data/textrecog/funsd_converter.py PATH/TO/funsd --nproc 4
 ```
+### ILST
+- Step1: Download dataset from [here](https://iiitaphyd-my.sharepoint.com/:f:/g/personal/minesh_mathew_research_iiit_ac_in/EtLvCozBgaBIoqglF4M-lHABMgNcCDW9rJYKKWpeSQEElQ?e=zToXZP)
+- Step2: Run the following codes
+```bash
+unzip -q IIIT-ILST.zip && rm IIIT-ILST.zip
+cd IIIT-ILST
+# rename files
+cd Devanagari && for i in `ls`; do mv -f $i `echo "devanagari_"$i`; done && cd ..
+cd Malayalam && for i in `ls`; do mv -f $i `echo "malayalam_"$i`; done && cd ..
+cd Telugu && for i in `ls`; do mv -f $i `echo "telugu_"$i`; done && cd ..
+# transfer image path
+mkdir imgs && mkdir annotations
+mv Malayalam/{*jpg,*jpeg} imgs/ && mv Malayalam/*xml annotations/
+mv Devanagari/*jpg imgs/ && mv Devanagari/*xml annotations/
+mv Telugu/*jpeg imgs/ && mv Telugu/*xml annotations/
+# unify postfix
+cd imgs
+ls -1 *.jpeg | xargs -n 1 bash -c 'convert "$0" "${0%.jpeg}.jpg"'
+rm *jpeg
+# remove unnecessary files
+cd ..
+rm -rf Devanagari && rm -rf Malayalam && rm -rf Telugu && rm READEME.txt
+cd ../../
+```
+- Step3: Generate `train_label.txt` and `test_label.txt` and crop images using 4 processes with following command (add `--preserve-vertical` if you wish to preserve the images containing vertical texts). Besides, the orginal dataset doesn't have test set. And specific `--test_ratio` to split the dataset.
```

Comment:
```suggestion
- Step3: Generate `train_label.txt` and `test_label.txt` and crop images using 4 processes with the following command (add `--preserve_vertical` if you wish to preserve the images containing vertical texts). Since the original dataset doesn't have a test set, you may specify `--test_ratio` to split the dataset. 20% of the data are left out as the test set in this example.
```